src/main/scala/scopt/options.scala

```diff
@@ -354,19 +354,24 @@ abstract case class OptionParser[C](programName: String) {
   }
   def optionsForRender: List[OptionDef[_, C]] = {
     val unsorted = options filter { o => o.kind != Head && o.kind != Check && !o.isHidden }
-    val (unseen, xs) = unsorted partition {_.hasParent} match {
+    val (remaining, sorted) = unsorted partition {_.hasParent} match {
```

Comment:
I renamed some of the variables to make it easier to understand what this code is doing